.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
```

Comment:
Im curious what this do?

Reply:
for dev containers, dealing with line endings from windows to linux.
https://code.visualstudio.com/docs/devcontainers/tips-and-tricks#_resolving-git-line-ending-issues-in-containers-resulting-in-many-modified-files